### ISSUE TITLE
Any plans to release the code?

### ISSUE THREAD

**Issue by chenhaomingbob** [hustvl/AziNorm#4]
Hi, thank you for your great work! Any plans to release the code?

**Comment by outsidercsy** [comment]
Of course. But these days I'm heavily involved in some other projects and have to delay the release. Sorry for that. The code will be available after the main conference of CVPR 22.

**Comment by kushagra97pal** [comment]
> Of course. But these days I'm heavily involved in some other projects and have to delay the release. Sorry for that. The code will be available after the main conference of CVPR 22.

when will you release the code..
